docs/en/latest/plugins/response-rewrite.md

```diff
@@ -32,13 +32,20 @@ response rewrite plugin, rewrite the content returned by the upstream as well as
+| body            | string  | optional    |         |                 | New `body` to client, and the content-length will be reset too.                                                                                                                                                                                                                                                         |
+| body_base64     | boolean | optional    | false   |                 | Identify if `body` in configuration need base64 decoded before rewrite to client.                                                                                                                                                                                                                                       |
+| headers         | object  | optional    |         |                 | Set the new `headers` for client, can set up multiple. If it exists already from upstream, will rewrite the header, otherwise will add the header. You can set the corresponding value to an empty string to remove a header. The value can contain Nginx variables in `$var` format, like `$remote_addr $balancer_ip`. |
+| vars            | array[] | optional    |         |                 | A DSL to evaluate with the given ngx.var. See `vars` [lua-resty-expr](https://github.com/api7/lua-resty-expr#operator-list). if the `vars` is empty, then all rewrite operations will be executed unconditionally.                                                                                                      |
+| filters         | array[] | optional    |         |                 | A group of filters that modify response body by replacing one specified string by another.                                                                                                                                                                                                                              |
+| filters.regex   | string  | required    |         |                 | match pattern on response body.                                                                                                                                                                                                                                                                                         |
+| filters.scope   | string  | optional    | "once"  | "once","global" | substitution range.                                                                                                                                                                                                                                                                                                     |
```

Comment:
It would be a good idea to explain the difference between `once` and `global`.

Reply:
Yep, had added their usage.